### DISCUSSION TITLE
Problem when asking to restart the remote device

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#6556] [General]
### Bug Description

If you ask to restart the remote device and there is another windows session open for another user, the confirmation dialogue appears on the remote computer, while on the RustDesk client it does not. 
Worse, you can no longer access the remote computer.

### How to Reproduce

On remote site 
Login to Windows with an account
Choose Change User and login with another account

On local site
From menu choose "Restart remote device"

### Operating system(s) on local side and remote side

Windows 10 -> Windows 11

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

On local site confirm to restart remote device
![1](https://github.com/rustdesk/rustdesk/assets/8618744/a85558a8-938b-47a2-8c45-2e595eb5e3f6)

On local site nothing on screen
![2](https://github.com/rustdesk/rustdesk/assets/8618744/48d75b8b-da98-443f-84cb-30879ab97473)

On remote site there is confirm to restart device
![3](https://github.com/rustdesk/rustdesk/assets/8618744/d34d117e-455f-4f25-bd4d-7b824123e4c3)

**Comment by rustdesk** [comment]
Is the controlled side installed?

**Reply by pdifeo** [reply]
of course

**Reply by rustdesk** [reply]
@21pages verify this please. 

If should we use the `force_reboot`? https://github.com/risoflora/system_shutdown/blob/4d93e5e8c86ab94a1b7073b09b2b5b83096a35a8/src/windows.rs#L144

**Reply by rustdesk** [reply]
Let's try out `force_reboot` first for this case. If `force_reboot` not work, please try out Anydesk, if it works, we need to find correct api.

**Reply by pdifeo** [reply]
Just for your information. I'm sure that on TeamViewer is working. Tested rigth now

**Reply by rustdesk** [reply]
Fix merged https://github.com/rustdesk/rustdesk/pull/6557
